go/cmd/dolt/commands/sqlserver/sqlserver.go

```diff
@@ -46,8 +46,46 @@ const (
 
 var sqlServerDocs = cli.CommandDocumentationContent{
 	ShortDesc: "Start a MySQL-compatible server.",
-	LongDesc:  `Start a MySQL-compatible server which can be connected to by MySQL clients.`,
+	LongDesc: "In it's default configuration, starts a MySQL-compatible server which allows only one user connection at" +
+		"a time to the dolt repository in the current directory. Any edits made through this server will be automatically " +
+		"reflected in the working set.  This behavior can be modified using a yaml configuration file passed to the server " +
+		"via --config <file>, or by using the supported switches and flags to configure the server directly on the " +
+		"command line (If --config <file> is provided all other command line arguments are ignored).\n" +
+		"\n" +
+		"This is an example yaml configuration file showing all supported items and their default values:\n" +
+		"\n" +
+		"\n" +
+		"{{.EmphasisLeft}}" + serverConfigAsYAMLConfig(DefaultServerConfig()).String() + "{{.EmphasisRight}}\n" +
+		"\n" +
+		"\n" +
+		"SUPPORTED CONFIG FILE FIELDS:\n" +
+		"\n" +
+		"log_level - Level of logging provided. Options are: `trace', `debug`, `info`, `warning`, `error`, and `fatal`.\n" +
+		"\n" +
+		"behavior.read_only - If true database modification is disabled.\n" +
+		"\n" +
+		"behavior.autocommit - If true write queries will automatically alter the working set.  When working with autocommit " +
+		"enabled it is highly recommended that listener.max_connections be set to 1 as concurrency issues will arise otherwise.\n" +
+		"\n" +
+		"user.name - The username that connections should use for authentication.\n" +
+		"\n" +
+		"user.password - The password that connections should use for authentication.\n" +
+		"\n" +
+		"listener.host - The host address that the server will run on.  This may be an `localhost` or an IPv4 or IPv6 address.\n" +
+		"\n" +
+		"listener.port - The port that the server should listen on.\n" +
+		"\n" +
+		"listener.max_connections - The number of simultaneous connections that the server will accept.\n" +
+		"\n" +
+		"listener.read_timeout_millis - The number of milliseconds that the server will wait for a read operation.\n" +
+		"\n" +
+		"listener.write_timeout_millis - The number of milliseconds that the server will wait for a write operation.\n" +
+		"\n" +
+		"\n" +
```

Comment:
Should document the databases field as well

Reply:
done